### ISSUE TITLE
Missing TS definitions

### ISSUE THREAD

**Issue by dboakes** [inovua/reactdatagrid#15]
- what edition are you using - community
- version for `@inovua/reactdatagrid-community` - 4.0.17

Relevant code or config

```javascript
<ReactDataGrid
      idProperty="id"
      columns={columns}
      dataSource={dataSource}
      onSortInfoChange={handleSort}
      limit={10}
      // @ts-ignore
      onLimitChange={handlePageSize}
      // @ts-ignore
      onSkipChange={handlePageChange}
      sortable
      allowUnsort={false}
      style={gridStyle}
      pagination='remote' />
```

What you did: 
Trying to do server site sorting/filtering/pagination, but typescript seems to be missing some definitions

What happened:
```javascript
// @ts-ignore
onLimitChange={handlePageSize}
// @ts-ignore
onSkipChange={handlePageChange}
```
If I use @ts-ignore, both onLimitChange and onSkipChange work as expected, so its just the definitions missing. 

Reproduction repository:
[Codesandbox](https://codesandbox.io/s/upbeat-dust-1u8or?file=/src/App.tsx)

Problem description:
I don't want to have to use ts-ignore to make the program work.

**Comment by inovua-admin** [comment]
Thanks for reporting, we'll include those in the next version

**Comment by marquetd** [comment]
I think "filterEditor" for "TypeColumn" is missing.

**Comment by inovua-admin** [comment]
fixed in `4.0.19`
